### ISSUE TITLE
problems search reset after press back button

### ISSUE THREAD

**Issue by demidenko** [xorum-io/codeforces_watcher#135]
android 10
1) open problems tab
2) type search text
3) open problem
4) press back (android bottom button not top bar arrow)
5) view is recreated and text cleared

**Comment by MatyashDen** [comment]
@yevhenii-kanivets @bogdan-evtushenko 
Anyone succeed to reproduce it?

**Comment by yevhenii-kanivets** [comment]
@MatyashDen yes, it's easily reproduced by `press back (android bottom button not top bar arrow)`.
